apiclient/vote.go

```diff
@@ -26,12 +29,12 @@ import (
 // KeyType is the type of the key used when the census was created. It can be
 // either models.ProofArbo_ADDRESS or models.ProofArbo_PUBKEY (default).
 type VoteData struct {
-	Choices    []int
-	ElectionID types.HexBytes
+	Choices      []int
+	ElectionID   types.HexBytes
+	VotingWeight *big.Int
 
 	ProofMkTree *CensusProof
 	ProofCSP    types.HexBytes
-	ProofZkTree *CensusProofZk
 }
 
 // Vote sends a vote to the Vochain. The vote is a VoteData struct,
```

Comment:
I think we should document on the description of this method the behavior of VotingWeight (how can it be used and what happens if it is nil).